eclair-core/src/main/resources/reference.conf

```
@@ -15,6 +15,7 @@ eclair {
     password = "" // password for basic auth, must be non empty if json-rpc api is enabled
   }
 
+  disable-db-backup = false // disable the automatic sqlite db backup; do not change this unless you know what you are doing
```

Did you do the double negative on purpose? I think it's better to avoid it

```suggestion
  enable-db-backup = true // enable the automatic sqlite db backup; *do not change this unless you know what you are doing*
```

---

I wanted to make it more explicit that the default behavior is to have it `enabled`, and we only provide an option to `disable` it. But it's nit, I don't have a strong opinion.

---

Yeah I figured you had that in mind. But even knowing it, I had to think twice before deciding whether backups would be on or off by default. I think the other way around is better.